frontend/src/App.js

```diff
@@ -1,12 +1,43 @@
-
 import './App.css';
+import Header from "./components/Header";
+import NavigationBar from "./components/NavigationBar";
+import {BrowserRouter as Router, Route, Switch} from "react-router-dom";
+import NewDepotPage from "./pages/NewDepotPage";
+import EditDepotPage from "./pages/EditDepotPage";
+import PlayPage from "./pages/PlayPage";
+import HomePage from "./pages/HomePage";
+import useStocks from "./hooks/useStocks";
+import SwitchDepotPage from "./pages/SwitchDepotPage";
+import {CssBaseline} from "@mui/material";
 
 function App() {
-  return (
-    <div>
-      Hello World!
-    </div>
-  );
+    const {stocks} = useStocks()
+    return (
+        <div>
+            <Router>
+                <CssBaseline/>
+                <Header/>
+                <Switch>
+                    <Route path="/new">
+                        <NewDepotPage/>
```

Comment:
i think the page name is better without "Page" at the end -> Clean Code Explained (Avoid Disinformation)

Reply:
I will rethink the hole passage with my pages and the routes, thanks for your inspiration.